security/pkg/pki/ra/k8s_ra.go

```diff
@@ -125,14 +125,14 @@ func (r *KubernetesRA) SignWithCertChain(csrPEM []byte, certOpts ca.CertOpts) ([
 	var possibleRootCert, rootCertFromMeshConfig, rootCertFromCertChain []byte
 	certSigner := r.certSignerDomain + "/" + certOpts.CertSigner
 	if len(r.GetCAKeyCertBundle().GetRootCertPem()) == 0 {
-		rootCertFromCertChain, err = util.FindRootCertFromCertificateChainBytes(cert)
-		if err != nil {
```

Comment:
we should  just log the info that no root cert found in cert chain instead of return error out.

Reply:
Should we check the MeshConfig for the root cert before checking the cert chain? If not, I'll update the comment for the method.

Reply:
we already set final root cert as the one from meshconfig if both exists,   see below line 136.

Reply:
Got it! Reverted change to the original ordering of checking the cert chain and then the MeshConfig.